style.css

```diff
@@ -1066,219 +733,48 @@ ul li {
-  }
-
-  .css {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 15px;
-    line-height: 20px;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    gap: 16px;
+    justify-content: space-around;
     align-items: flex-start;
+    width: 100%;
+    height: 106.72px;
+    padding-left: 2%;
     color: #253858;
-    width: 122px;
-    height: 120px;
     background: #f7f7f9;
     border-radius: 8px;
   }
 
-  #framework {
-    position: absolute;
-    top: 310px;
-  }
-
-  .arrow1 {
-    position: absolute;
-    height: 24px;
-    left: 636px;
-    right: 78px;
-    top: 310px;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 20px;
-    line-height: 24px;
-    display: flex;
-    align-items: center;
-    color: #344563;
-  }
-
-  #skills {
-    position: absolute;
-    top: 392px;
-  }
-
-  .arrow2 {
-    position: absolute;
-    height: 24px;
-    left: 636px;
-    right: 78px;
-    top: 392px;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 20px;
-    line-height: 24px;
-    display: flex;
-    align-items: center;
-    color: #344563;
-  }
-
-  .separator1 {
-    position: absolute;
-    width: 684px;
-    height: 1px;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    top: 363px;
-    border-radius: 0;
-    background: #dfe1e6;
-  }
-
-  .separator2 {
-    position: absolute;
-    width: 684px;
-    height: 1px;
-    left: 0;
-    right: 0;
-    top: 441px;
-    bottom: 0;
-    background: #dfe1e6;
-  }
+  /* Contact me section */
 
   .contactme {
-    position: absolute;
-    width: 1440px;
-    height: 846px;
-    top: 4362px;
-    background-color: #6070ff;
-    border-radius: 10% 0 0 0/10% 0 0 0;
-  }
-
-  .contacttxt {
-    width: 801px;
-    height: 56px;
-    font-family: Poppins, sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 40px;
-    line-height: 52px;
-    display: flex;
-    align-items: center;
-    text-align: center;
-    color: #fff;
-    margin-top: 60px;
-    margin-left: 330px;
-  }
-
-  .paragraph {
-    width: 569px;
-    height: 102px;
-    font-family: Poppins, sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 20px;
-    line-height: 28px;
-    text-align: center;
-    color: #ebebff;
-    margin-top: 60px;
-    margin-left: 420px;
+    background-image: url('img/contact-form-bg.png');
   }
 
-  .inputtxt1 {
-    position: absolute;
-    height: 48px;
-    width: 447px;
-    margin-left: auto;
-    margin-right: auto;
-    top: 340px;
-    bottom: 0;
-    background: #fff;
-    border: 1px solid #cfd8dc;
-    border-radius: 2px;
+  .contactme > h2 {
+    margin: auto 25%;
+    padding: 4%;
   }
 
-  .inputtxt2 {
-    position: absolute;
-    height: 48px;
-    width: 447px;
-    margin-left: auto;
-    margin-right: auto;
-    top: 420px;
-    bottom: 0;
-    background: #fff;
-    border: 1px solid #cfd8dc;
-    border-radius: 2px;
+  .contactme > p {
+    margin: auto 15%;
```

Comment:
- [ ] Kindly reduce the width of your form paragraphy in order to match the figma design. Take a look :point_down: 

###  Your Design
![your-form](https://github.com/ntimanu/Portfolio/assets/26597432/023e0d70-65ae-480b-85e3-ea50b73e6aab)

### Figma Design
![temp-form](https://github.com/ntimanu/Portfolio/assets/26597432/ca9283f9-389e-4d47-80fa-c71dfe8e2b86)

Reply:
Done